code/drasil-lang/Language/Drasil/Reference.hs

```diff
@@ -271,8 +273,37 @@ instance Referable Contents where
 uidSort :: HasUID c => c -> c -> Ordering
 uidSort = compare `on` (^. uid)
 
+compareAuthYearTitle :: (HasFields c) => c -> c -> Ordering
+compareAuthYearTitle c1 c2
+  | comparePeople (getAuthor c1) (getAuthor c2) /= Nothing = fromJust $ comparePeople (getAuthor c1) (getAuthor c2)
+  | (getYear c1)  /= (getYear c2)  = (getYear c1)  `compare` (getYear c2)
+  | (getTitle c1) /= (getTitle c2) = (getTitle c1) `compare` (getTitle c2)
+  | otherwise                      = error "Couldn't sort authors"
+
+getAuthor :: (HasFields c) => c -> People
+getAuthor c = maybe (error "No author found") (\(Author x) -> x) (find (isAuthor) (c ^. getFields))
+  where isAuthor :: CiteField -> Bool
+        isAuthor (Author _) = True
+        isAuthor _          = False 
+
+getYear :: (HasFields c) => c -> Int
+getYear c = maybe (error "No year found") (\(Year x) -> x) (find (isYear) (c ^. getFields))
+  where isYear :: CiteField -> Bool
+        isYear (Year _) = True
+        isYear _        = False
+
+getTitle :: (HasFields c) => c -> String
+getTitle c = getStr $ maybe (error "No title found") (\(Title x) -> x) (find (isTitle) (c ^. getFields))
```

Comment:
you don't need () around `isTitle`.

Reply:
I'm assuming this applies to `isAuthor` and `isYear` as well

Reply:
Right, function names do not need to be wrapped in ().